plugins/@grouparoo/iterable/src/lib/export/destinationMappingOptions.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  DestinationMappingOptionsMethod,
+  DestinationMappingOptionsResponseTypes,
+} from "@grouparoo/core";
+import { connect } from "./../connect";
+
+export const destinationMappingOptions: DestinationMappingOptionsMethod = async ({
+  appOptions,
+}) => {
+  const client = await connect(appOptions);
+  const required = getRequiredFields();
+  const known = await getUserFields(client);
+
+  return {
+    labels: {
+      property: {
+        singular: "Iterable User Field",
+        plural: "Iterable User Fields",
+      },
+      group: {
+        singular: "Iterable List",
+        plural: "Iterable Lists",
+      },
+    },
+    properties: {
+      required: required,
+      known: known,
+      allowOptionalFromProperties: true,
+    },
+  };
+};
+
+export const getRequiredFields = (): Array<{
+  key: string;
+  type: DestinationMappingOptionsResponseTypes;
+}> => {
+  return [{ key: "email", type: "email" }];
+};
+
+const mapTypesFromIterableToGrouparoo = (iterableType) => {
+  const map = {
+    string: "string",
+    long: "integer",
+    date: "date",
+    boolean: "boolean",
+  };
+  const grouparooType = map[iterableType];
+  if (grouparooType) {
+    return grouparooType;
+  }
+  return null;
+};
+
+export const getUserFields = async (
+  client: any
+): Promise<
+  Array<{
+    key: string;
+    type: DestinationMappingOptionsResponseTypes;
+    important?: boolean;
+  }>
+> => {
+  const fields = await client.users.getFields();
+  const out = [];
+
+  for (const [key, value] of Object.entries(fields.fields)) {
+    if (value !== "object" && key !== "email") {
+      const type: DestinationMappingOptionsResponseTypes = mapTypesFromIterableToGrouparoo(
+        value
+      );
+      if (type) {
+        out.push({ key, type, important: true });
```

Comment:
they probably aren't all `important`. We probably need a whitelist with these in it:
name
phoneNumber
userId
signupDate

I don't think we want the ones with `"."` to be `important`
Maybe that's a way to do it.

I think we also want to leave out these ones:
emailListIds
unsubscribedChannelIds
unsubscribedMessageTypeIds